source/fundamentals/data-formats/document-data-format-json.txt

```diff
@@ -0,0 +1,45 @@
+==========================
+Document Data Format: JSON
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+- Describe Extended JSON
+- Question here: what is the difference between toJson() and parse()?
```

Comment:
toJson() converts _to_ JSON, parse() converts _from_ JSON

Reply:
@ccho-mongodb please respond.

Reply:
Updated, thanks. Sorry, not sure how I missed this one.